src/fsharp/FSharp.Core/Linq.fs

```diff
@@ -302,30 +319,41 @@ module LeafExpressionConverter =
     let (|CheckedMinusQ|_|)    = (|SpecificCallToMethod|_|) (methodhandleof (fun (x,y) -> Checked.( - ) x y))
     let (|CheckedMultiplyQ|_|) = (|SpecificCallToMethod|_|) (methodhandleof (fun (x,y) -> Checked.( * ) x y))
 
-    let (|ConvCharQ|_|)        = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> char x))
-    let (|ConvDecimalQ|_|)     = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> decimal x))
-    let (|ConvFloatQ|_|)       = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> float x))
-    let (|ConvFloat32Q|_|)     = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> float32 x))
-    let (|ConvSByteQ|_|)       = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> sbyte x))
-    let (|ConvInt16Q|_|)       = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> int16 x))
-    let (|ConvInt32Q|_|)       = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> int32 x))
-    let (|ConvIntQ|_|)         = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> int x))
-    let (|ConvInt64Q|_|)       = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> int64 x))
-    let (|ConvByteQ|_|)        = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> byte x))
-    let (|ConvUInt16Q|_|)      = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> uint16 x))
-    let (|ConvUInt32Q|_|)      = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> uint32 x))
-    let (|ConvUInt64Q|_|)      = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> uint64 x))
+    let (|ConvCharQ|_|)        = (|SpecificCallToMethod|_|) (methodhandleof (fun x -> Operators.char x))
```

Comment:
Throughotu this file "Operators." has been included to give important code clarification as part of the bug fix as to which operator implementation is being called